src/sentry/integrations/utils/code_mapping.py

```diff
@@ -399,13 +397,21 @@ def convert_stacktrace_frame_path_to_source_path(
     )
 
     if stacktrace_path and stacktrace_path.startswith(code_mapping.stack_root):
-        return stacktrace_path.replace(stack_root, code_mapping.source_root, 1)
+        return (
+            stacktrace_path.replace(stack_root, code_mapping.source_root, 1)
+            .lstrip("/")
+            .replace("\\", "/")
+        )
 
     # Some platforms only provide the file's name without folder paths, so we
     # need to use the absolute path instead. If the code mapping has a non-empty
     # stack_root value and it matches the absolute path, we do the mapping on it.
     if frame.abs_path and frame.abs_path.startswith(code_mapping.stack_root):
-        return frame.abs_path.replace(stack_root, code_mapping.source_root, 1)
+        return (
+            frame.abs_path.replace(stack_root, code_mapping.source_root, 1)
```

Comment:
We do `.replace("\\", "/")` _after_ we replace the stack root with the source root in the abs_path because the stack_root maintains the slash type from the stacktrace path it was derived from. 

This is what acuse the issue before: on lines 390-391 of the previous version, we replaced the backslashes with slashes in the stack root _before_ we applied the code mapping to the abs_path. But since the abs_path does not also have this transformation, it will never apply correctly. 

(There's **a lot** of variables and things to keep track of here, lmk if some more clarification into my thinking would be helpful)

Reply:
I think this makes sense. It _does_ break some existing functionality, if someone had a stack_root with backslashes but had code with front slashes. But that doesn't sound very likely. You could be super safe and run the replacement on everything to avoid that if you wanted to

Reply:
Yeah that definitely is pretty unlikely, and I would argue that if that were the case, then it would make more sense to generate an entirely new code mapping for stacktrace frame with forward slashes – there are some nuances that make it not just a 1 to 1 translation if that makes sense